src/components/groups/SideBar.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from "react";
+import MembersCard from "./MembersCard";
+import {
+  useAutoFillGroupMutation,
+  useGetGroupByIdQuery,
+} from "<@>/store/groups/groups-api";
+import ConfrimationCard from "./ConfirmAutoFill";
+import Modal from "../common/Modal";
+import { AiOutlineCloseCircle } from "react-icons/ai";
+import Loading from "../common/Loading";
+import FetchingError from "../common/FetchingError";
+
+interface SidebarProps {
+  id: string;
+  setSelectedGroup: (value: string) => void;
+}
+
+const Sidebar: React.FC<SidebarProps> = ({ id, setSelectedGroup }) => {
+  const { data, isLoading, error } = useGetGroupByIdQuery(id);
+  const [autoFillGroup, { isLoading: isFilling }] = useAutoFillGroupMutation();
+  const [showPopup, setShowPopup] = useState<Boolean>(false);
+  const [searchValue, setSearchValue] = useState("");
+
+  const handleAutofill = async (id: any) => {
```

Comment:
If you don't need to perform any actions after autofill, you can remove the 'async/await' keywords.